notebooks/en/phoenix_observability_on_hf_spaces.ipynb

```diff
@@ -0,0 +1,594 @@
+{
```

Comment:
<p>The "Note" can be replaced with <code>&gt; [!TIP]</code> to use the doc-builder's syntax (same for the Note above)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/227/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Good call, thanks! Just updated all of them</p>